skills/declarative/calendarSkill/dialogs/SearchEventsDialog/language-understanding/en-us/SearchEventsDialog.en-us.lu

```diff
@@ -40,4 +40,10 @@
 - not quite
 - not right now
 - not that
-- oh no
\ No newline at end of file
+- oh no
+
+@ list order=
```

Comment:
Is this to add on to the prebuild OrdinalV2 entity? If so, we could combine them into one property using coalesce().

Reply:
changed to OrdinalV2